discord/client.py

```diff
@@ -950,8 +949,7 @@ async def change_presence(self, *, activity=None, status=None, afk=False):
         """
 
         if status is None:
-            status = 'online'
-            status_enum = Status.online
+            return
```

Comment:
This isn't the correct solution to the problem, and would also break setting the activity.